db/dump.py

```diff
@@ -20,6 +20,9 @@
 import os
 from sqlalchemy import text
 
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
can we use the flask logger here? (I think the management tasks will be running inside an app context, right?)